examples/src/main/java/com/mapbox/navigation/examples/ui/BuildingFootprintHighlightActivityKt.kt

```diff
@@ -138,29 +139,11 @@ class BuildingFootprintHighlightActivityKt : AppCompatActivity(), OnNavigationRe
         return instructions!!
     }
 
-    override fun allowRerouteFrom(offRoutePoint: Point?): Boolean {
-        // Not needed in this example
-        return true
+    override fun onStopArrival(routeLegProgress: RouteLegProgress) {
+        // todo: no op
```

Comment:
What does `// todo: no op` mean? Can we leave a clearer message? What's the TODO? 

For now, maybe just `// Empty because this callback's not needed in this example`